package.json

```diff
@@ -41,11 +41,12 @@
     "axios": "^1.6.1"
   },
   "devDependencies": {
+    "@faker-js/faker": "^8.4.1",
```

Comment:
I'll take care of pushing this change to the template [here](https://github.com/onfido/onfido-openapi-spec/blob/master/generators/typescript-axios/templates/package.mustache#L47-L52)

Reply:
Rebase already done.